wiki/articles/maven-integration.md

```diff
@@ -1,29 +1,25 @@
 ---
 title: Maven integration
 ---
-Instead of using Gradle to setup your libGDX project, you can also use Maven. However, **this is _not_ recommended!** When issues arise, getting support will most certainly prove very difficult!
+*last update: 13/03-2024 (DDMMYYYY)*
```

Comment:
I would only add a "last updated", if all infos are really up to date there, which it seems to not be.

Reply:
That's fair. Do you know when the latest update is from?

Reply:
As long as the whole page isn't up to date, we should keep a disclaimer. The page is pretty much unchanged since its creation about 11 years ago.